pkg/services/libraryelements/api.go

```diff
@@ -35,12 +36,15 @@ func (l *LibraryElementService) createHandler(c *models.ReqContext, cmd CreateLi
 
 // deleteHandler handles DELETE /api/library-elements/:uid.
 func (l *LibraryElementService) deleteHandler(c *models.ReqContext) response.Response {
-	err := l.deleteLibraryElement(c.Req.Context(), c.SignedInUser, web.Params(c.Req)[":uid"])
+	id, err := l.deleteLibraryElement(c.Req.Context(), c.SignedInUser, web.Params(c.Req)[":uid"])
 	if err != nil {
 		return toLibraryElementError(err, "Failed to delete library element")
 	}
 
-	return response.Success("Library element deleted")
+	return response.JSON(200, util.DynMap{
+		"message": "Library element deleted",
+		"id":      id,
+	})
```

Comment:
I thought we should avoid using util.DynMap but maybe I'm thinking about something else?

Reply:
I didn't know but it makes sense, especially when adding test! I created a structure for that, not sure about the name 🤔

Reply:
DeleteLibraryElementResult or LibraryElementDeleteResult?